### DISCUSSION TITLE
Tag-Index should show snippets of inline-tags.

### DISCUSSION THREAD

**Discussion by j-steinbach** [srid/emanote#208] [Ideas]
There are two kinds of tags: note-tags and inline-tags. The former is declared for a whole note, for example as frontmatter tag

```YAML
tags:
- some tag
- more tags
```

The inline-tags, however, are declared inline (`#d'oh`). 

At the moment Emanote in the [Tag Overview](https://note.ema.srid.ca/-/tags/emanote/dev) shows only the "big" note. This is not very useful for inline tags. 

I propse that the Tag Overview shows the the exact location of the inline tag and the line before and after. (Similar to `rg -B1 -A1`).

```
<filename>
----
<line before>
The inline-tags, however, are declared inline (`#d'oh`). 
<line after>
```

**Comment by srid** [comment]
See the unchecked task "Add backlink context to tag index pages (display the paragraph in which the inline tag is used)" in  https://github.com/srid/emanote/discussions/50
